packaging/aur/atlauncher/PKGBUILD

```diff
@@ -12,7 +12,7 @@ modpacks easily and quickly."
 arch=('any')
 url="https://github.com/ATLauncher/ATLauncher"
 license=('GPL3')
-depends=('java-runtime>=8' 'openal')
+depends=('java-runtime>=17' 'openal')
 makedepends=('java-environment>=8')
```

Comment:
Might be good to update this as well, cause once we go to Java 17, users will need it to build the application